models/semantic_segmentation/DUC/README.md

```diff
@@ -0,0 +1,72 @@
+
+### Input 
+Since the model is trained on the cityscapes dataset which contains images of urban street scenes, the input should preferably be an image of a street scene to produce best results. There are no constraints on the size of the image. The example in the inference notebook is done using a png image.
+
+### Preprocessing
+The DUC layer has an effect of partitioning the image into d^2 subparts (d = downsampling rate). This is why the input image is extrapolated with a small border in order to obtain an accurate reshaped image after the DUC layer. After this the image is normalized using mean subtraction. Check [duc-preprocess.py](duc-preprocess.py) for code.
+
+### Output
+The output of the network is a tensor of shape (1 X `label_num` X `H`*`W`) where `H` and `W` are the height and width of the output segmented map.
+
+### Postprocessing
+The output tensor is reshaped and resized to give the softmax map of shape (`H` X `W` X `label_num`). The raw label map is computed by doing an argmax on the softmax map which is used to generate the colorized segmented images. Check [duc-postprocess.py](duc-postprocess.py) for code.
+
+<!--
+To do quick inference with the model, check out [Model Server](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md/#arcface-resnet100_onnx).
+-->
+
+## <a name="dset"></a>Dataset
+Cityscapes dataset is used for training and validation. It is a large dataset that focuses on semantic understanding of urban street scenes. It contains 5000 images with fine annotations across 50 cities, different seasons, varying scene layout and background. There are a total of 30 categories in the dataset of which 19 are included for training and evaluation. The training, validation and test set contains 2975, 500 and 1525 fine images, respectively.
+
+### Download
+First, go to the [Cityscapes download page](https://www.cityscapes-dataset.com/downloads/) and register for an account (login if account already made). Next, find and download the following two files:
+
+|Filename                 | Size  | Details|
+|-------------------------|:------|:-------|
+|leftImg8bit_trainvaltest.zip| 11 GB| train/val/test images|
+|gtFine_trainvaltest.zip  | 241 MB| fine annotations for train and val sets|
+
+### Setup
+* Unpack the zip files into folders `leftImg8bit_trainvaltest` and `gtFine_trainvaltest`.
+* Use the path to the train/val folders inside these folders for training/validation.
+* Use the files [val.lst](https://s3.amazonaws.com/onnx-model-zoo/duc/val.lst) and [train.lst](https://s3.amazonaws.com/onnx-model-zoo/duc/train.lst) containing image lists required during validation and training respectively.
+
+Please note that the dataset is under copyright. Refer to the [citation](https://www.cityscapes-dataset.com/citation/) page for details.
+
+## Validation accuracy
+The accuracies obtained by the models on the validation set are mentioned above and they match with those mentioned in the paper.
```

Comment:
Mention that the metric definition is mentioned below in the validation section

Reply:
same as above